src/Util/DigitizeUtil/DigitizeUtil.js

```diff
@@ -0,0 +1,81 @@
+import OlObservable from 'ol/observable';
+
+/**
+ * This class provides some static methods which might be helpful when working
+ * with redlining functions.
+ *
+ * @class DigitizeUtil
+ */
+class DigitizeUtil {
+
+  /**
+   * Moves / translates an `OlFeature` to the given `pixel` delta in
+   * in the end with given `duration` in ms, using the given style,
+   * calling a `doneFn`.
+   *
+   * @param {OlMap} map An OlMap.
```

Comment:
`OlMap` -> `ol.Map`

Reply:
Why?

Reply:
To get in sync with the `ol` API and to harmonize with the method documentation overall. But it's still non-blocking…